src/components/About/AboutUIModal.tsx

```diff
@@ -66,7 +66,7 @@ class AboutUIModal extends React.Component<AboutUIModalProps, AboutUIModalState>
     if (config().about && config().about.website) {
       return (
         <div>
-          <a href={config().about.website.url} target="_blank">
+          <a href={config().about.website.url} target="_blank" rel="noopener noreferrer">
```

Comment:
This` rel="noopener noreferrer"` is new to me. nice.

Reply:
I learnt this yesterday too :)